styles/rrt.go

```diff
@@ -6,13 +6,27 @@ import (
 
 // Rrt style.
 var Rrt = Register(chroma.MustNewStyle("rrt", chroma.StyleEntries{
-	chroma.Comment:        "#00ff00",
-	chroma.NameFunction:   "#ffff00",
-	chroma.NameVariable:   "#eedd82",
-	chroma.NameConstant:   "#7fffd4",
-	chroma.Keyword:        "#ff0000",
-	chroma.CommentPreproc: "#e5e5e5",
-	chroma.LiteralString:  "#87ceeb",
-	chroma.KeywordType:    "#ee82ee",
-	chroma.Background:     " bg:#000000",
+	chroma.CommentMultiline:    "#00ff00",
+	chroma.CommentPreproc:      "#e5e5e5",
+	chroma.CommentSingle:       "#00ff00",
+	chroma.CommentSpecial:      "#00ff00",
+	chroma.Comment:             "#00ff00",
```

Comment:
The token types are actually "hierarchical", in that if `CommentSpecial` is not defined it will use the style from `Comment`, and so on, so you should only need to override `CommentPreproc` here (as all the others have the same style).

Reply:
Ah I see, thanks for clarifying. I've changed this in the latest commit; the only duplicate is now `LiteralNumber` and `LiteralStringSymbol`. (Because those are tokens of different kinds.)